translations/ga/census_individual_gb_nir_ga.po

```diff
@@ -2,3312 +2,4784 @@ msgid ""
+#. Answer option
+msgctxt "What is your position in this establishment?"
 msgid "Staff or owner"
 msgstr "Ball foirne nó úinéir"
 
-#. answer-id: establishment-position-answer
-msgctxt "Answer for: What is your position in this establishment?"
+#. Answer option
+msgctxt "What is your position in this establishment?"
 msgid "Family member or partner of staff or owner"
 msgstr "Ball teaghlaigh nó páirtnéir bhall foirne nó an úinéara"
 
-#. answer-id: establishment-position-answer
-msgctxt "Answer for: What is your position in this establishment?"
+#. Answer option
+msgctxt "What is your position in this establishment?"
 msgid "Staying temporarily"
 msgstr "Ag fanacht go sealadach"
 
-#. answer-id: establishment-position-answer
-msgctxt "Answer for: What is <em>{person_name_possessive}</em> position in this establishment?"
+#. Answer option
+msgctxt "What is <em>{person_name_possessive}</em> position in this establishment?"
 msgid "Resident"
 msgstr "Cónaitheoir"
 
-#. answer-id: establishment-position-answer
-msgctxt "Answer for: What is <em>{person_name_possessive}</em> position in this establishment?"
+#. Answer option
+msgctxt "What is <em>{person_name_possessive}</em> position in this establishment?"
 msgid "Staff or owner"
 msgstr "Ball foirne nó úinéir"
 
-#. answer-id: establishment-position-answer
-msgctxt "Answer for: What is <em>{person_name_possessive}</em> position in this establishment?"
+#. Answer option
+msgctxt "What is <em>{person_name_possessive}</em> position in this establishment?"
 msgid "Family member or partner of staff or owner"
 msgstr "Ball teaghlaigh nó páirtnéir bhall foirne nó úinéara"
 
-#. answer-id: establishment-position-answer
-msgctxt "Answer for: What is <em>{person_name_possessive}</em> position in this establishment?"
+#. Answer option
+msgctxt "What is <em>{person_name_possessive}</em> position in this establishment?"
 msgid "Staying temporarily"
 msgstr "Ag fanacht go sealadach"
 
-#. answer-id: confirm-date-of-birth-answer
-msgctxt "Answer for: You are {age_in_years} years old. Is this correct?"
-msgid "No, I need to change my date of birth"
-msgstr "Níl, caithfidh mé mo dháta breithe a athrú"
-
-#. answer-id: confirm-date-of-birth-answer
-msgctxt "Answer for: {person_name} is {age_in_years} years old. Is this correct?"
-msgid "No, I need to change their date of birth"
-msgstr "Níl, caithfidh mé a ndáta breithe a athrú"
-
-#. answer-id: sex-answer
-msgctxt "Answer for: What is your sex?"
+#. Answer option
+msgctxt "You are {age} old. Is this correct?"
+msgid "Yes, I am {age} old"
+msgstr "Tá mé {age} bliain d'aois"
+
+#. Answer option
+msgctxt "You are {age} old. Is this correct?"
+msgid "No, I need to correct my date of birth"
+msgstr "Níl, ní mór dom mo dháta breithe a cheartú"
+
+#. Answer option
+msgctxt "<em>{person_name}</em> is {age} old. Is this correct?"
+msgid "Yes, {person_name} is {age} old"
+msgstr "Tá {person_name} {age} bliain d'aois"
```

Comment:
Same issue as household with repetition of the translation for year/years.

Reply:
changed

Reply:
changed